### PR TITLE
Insights are not showed on successful tests

### PR DESCRIPTION
**Pull request by napoles-mk**
### Briefing
Having a test executed successfully does not show insights for steps as accurate. For instance, a script step may throw an exception nevertheless the execution could be successful if snippet is not for verification. This behavior is expected but we want to show in the step the insight about the exception thrown.

### Github Task
https://github.com/muuklabs/prototype/issues/1551

### Test executed

- Executed test with failing snippet (success and failing test)
- Executed test with success snippet (success and failing test)
- Executed test with both together, failing and successful snippet
- Executed test with all steps successful
- Executed test with some steps with warnings

### Answer the following questions
**Does the PR have any dependency with BE or FE? If so, specify which PR**
No
**In case of a dependency, could you describe what would happen if not delivered together? (eg. FE will crash, code will not compile, etc)**
NA
**Did you update the extension? if that's the case did you update the extension on remote Pupp project?**
No
**Did you add a new lib package (package.json)? If so, did you update the open source spreadsheet?**
No
**Did you link the defect to this PR (Github only)**
NA, different repository: https://github.com/muuklabs/prototype/issues/1551
**If you are adding a new FE view or any significant UI change, have you shared the new UI to the team already?**
NA
**Did you include or update the unit tests?**
NA
**Did you execute the npm tests ?**
Yes
**Did you test it in both environments (dev/prod)?**
Dev/Testing stage
**If you update the routes... Did you update the sample.txt ?**
NA
**Did you execute the E2E ?**
Yes
**Did you create new E2E tests or modified existing ones?**
No
**Did you add the new tests or modifications to the spreadsheet?**
No
**What tags are you using?**
NA
**What test tags were affected?**
NA

### CODE REVIEW

File: CSSAnalyzer.py [hunk 1]
```diff
@@ -174,8 +174,10 @@ def parseImageSelector(selector, htmlElements, searchInfo, expectedIndex, tag):
    numberElementsFoundWithValue = 0
    expectedValue = searchInfo["value"]
 
-   for selector in htmlElements:
-      if(selector['src'] == expectedValue ):
+   for element in htmlElements:
+      # The element source sometimes comes with a relative path, so it starts with '../'.
+      # This is to remove the dots and compare against the end of expected value. 
+      if(element['src'] == expectedValue or expectedValue.endswith(element['src'][2:None])):
```
Comment (jesus-serranop) [inline CSSAnalyzer.py]: HI @napoles-mk , what test was failing because of this? This is not related at all to the JS snippet issues, right?

Comment (napoles-mk) [reply CSSAnalyzer.py]: Hey @jesus-serranop, this is an edge case I found. The analyzer was not working correctly when the element is an image. 
The use case forr this is a step where user clicks on an image to do a check.

Comment (jesus-serranop) [reply CSSAnalyzer.py]: Correct, just a small comment that we just released a fix for this today. All the executors were updated. You will have to update and then put this change again.

Comment (napoles-mk) [reply CSSAnalyzer.py]: Nice, thanks @jesus-serranop. I have merge. Besides the fix there was another part to check the relative paths from the src. That is still showed in my changes.
